src/import-service/handlers/importFileParser.js

```diff
@@ -0,0 +1,49 @@
+import { accessHeaders } from '../constants/headers';
+import csv from 'csv-parser';
+
+const AWS = require('aws-sdk');
+const BUCKET = 'import-files-halinapp';
+
+export const importFileParser =  ({ Records }) =>{
+    try{
+        const  s3 = new AWS.S3({region: 'eu-west-1'});
+              
+        Records.forEach( async(record)  => {
```

Comment:
you should not use the async function in such a manner!
Your sync handler will return the response before any file will be processed. It will start several async functions (one promise for one records) and then simply return the response. Right now try/catch will not capture any errors during the files processing